reservation_units/tasks.py

```diff
@@ -26,6 +29,80 @@ def _update_reservation_unit_pricings() -> None:
     logger.info(f"Updated {num_updated} reservation units with date {today}")
 
 
+@app.task(name="update_reservation_unit_pricings_tax_percentage")
+def update_reservation_unit_pricings_tax_percentage(
+    change_date: str,
+    current_tax: str,
+    future_tax: str,
+    ignored_company_codes: list[str] | tuple[str] = (),
+) -> None:
+    from reservation_units.models import ReservationUnitPricing, TaxPercentage
+
+    SentryLogger.log_message(
+        message="Task `update_reservation_unit_pricings_tax_percentage` started",
+        details=f"Task was run with "
+        f"change_date: {change_date}, "
+        f"current_tax: {current_tax}, "
+        f"future_tax: {future_tax}, "
+        f"ignored_company_codes: {ignored_company_codes}",
+        level="info",
+    )
+
+    change_date = date.fromisoformat(change_date)  # e.g. "2024-09-01"
+    current_tax, _ = TaxPercentage.objects.get_or_create(value=Decimal(str(current_tax)))
+    future_tax, _ = TaxPercentage.objects.get_or_create(value=Decimal(str(future_tax)))
```

Comment:
Aren't `current_tax` and `future_tax` already string? Or is the typing for the function wrong?

Reply:
Also, since you are overriding `current_tax` and `future_tax`, you lose some typing info since Python thinks they are still strings from the function definition.